dsc_lib/src/discovery/command_discovery.rs

```diff
@@ -37,31 +58,75 @@ impl CommandDiscovery {
         }
     }
 
+    fn get_resource_path_setting() -> Result<ResourcePathSetting, DscError>
+    {
+        if let Ok(v) = get_setting("resource_path") {
```

Comment:
In Rust code, we want to use snake_casing, but in the JSON itself, we want to use camelCasing.

Reply:
Updated.